commit ebd928224a138d4560dc0be3ef162162d62a9e43
Author: Carlos O'Donell <carlos@redhat.com>
Date:   Thu May 18 12:56:45 2023 -0400

    assert: Reformat Makefile.
    
    Reflow all long lines adding comment terminators.
    Sort all reflowed text using scripts/sort-makefile-lines.py.
    
    No code generation changes observed in binary artifacts.
    No regressions on x86_64 and i686.

diff --git a/assert/Makefile b/assert/Makefile
index 2bc9e2214e3e9a8b..24a9bdb96306ca08 100644
--- a/assert/Makefile
+++ b/assert/Makefile
@@ -22,7 +22,9 @@ subdir	:= assert
 
 include ../Makeconfig
 
-headers	:= assert.h
+headers := \
+  assert.h
+  # headers
 
 routines := \
   __assert \
@@ -30,7 +32,13 @@ routines := \
   assert \
   assert-perr \
   # routines
-tests := test-assert test-assert-perr tst-assert-c++ tst-assert-g++
+
+tests := \
+  test-assert \
+  test-assert-perr \
+  tst-assert-c++ \
+  tst-assert-g++ \
+  # tests
 
 ifeq ($(have-cxx-thread_local),yes)
 CFLAGS-tst-assert-c++.o = -std=c++11
@@ -38,7 +46,10 @@ LDLIBS-tst-assert-c++ = -lstdc++
 CFLAGS-tst-assert-g++.o = -std=gnu++11
 LDLIBS-tst-assert-g++ = -lstdc++
 else
-tests-unsupported += tst-assert-c++ tst-assert-g++
+tests-unsupported += \
+  tst-assert-c++ \
+  tst-assert-g++ \
+  # tests-unsupported
 endif
 
 include ../Rules
